### ISSUE TITLE
⛔ 🔗 Popup close icons have blue "selected" borders around them (Duplicate of #21466)

### ISSUE THREAD

**Issue by rblackfox** [elementor/elementor#25791]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

It seems that in a recent version update, popup close buttons now have a browser selection blue border around them.
![Screen Shot 2024-04-18 at 3 40 22 PM](https://github.com/elementor/elementor/assets/7915930/5d173616-ab9d-48c2-b6b7-05897bb0fec1)
![Screen Shot 2024-04-18 at 3 45 32 PM](https://github.com/elementor/elementor/assets/7915930/c478d9eb-499b-41bf-9156-32da1443d35d)

Happens in Chrome, Safari, Firefox & Brave.

### Steps to reproduce

1. In an incognito window visit a website page
2. When a popup opens the close button has a select border around it

### Expected behavior

Popup close buttons should now have the blue selection boarder around it (to my knowledge)

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.57 (Debian)
	MySQL version: mariadb.org binary distribution v10.6.17
	PHP Version: 8.2.17
	PHP Memory Limit: 56000M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.4.4
	WP Multisite: Yes
	Max Upload Size: 15 MB
	Memory limit: 8192M
	Max Memory limit: 56000M

== Theme ==
	Name: Hello Child
	Version: 1.0.1
	Author: 
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 3.0.1
	Parent Theme Author: Elementor Team


== Active Plugins ==

	Elementor
		Version: 3.21.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.18.3
		Author: Elementor.com


== Network Plugins ==

	WooCommerce
		Version: 8.7.0
		Author: Automattic

	Wordfence Security
		Version: 7.11.4
		Author: Wordfence


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active
	recaptcha_v3: Active
	woocommerce: Active


== Elementor Experiments ==
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Inactive
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Active
	Grid Container: Active
	Nested Elements Performance: Inactive by default
	Optimized Control Loading: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Inactive by default
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive
	Build with AI: Active by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Inactive by default
	Page Transitions: Inactive
	Notes: Inactive
	Form Submissions: Active
	Scroll Snap: Active
	Menu: Inactive by default
	Taxonomy Filter: Inactive by default

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by Daveden2** [comment]
It's not actually a bug. It's a focus outline designed to assist keyboard users. Ideally, it should only be visible to keyboard users. However, I believe iOS follows a different spec and forces the outline for all users.

Some individuals opt for a method that compromises keyboard accessibility by disabling the accessibility settings in the advanced tab of their popup settings.

I also previously devised a [solution](https://youtu.be/3pfPYhgmAZU). Alternatively, you could employ CSS to simply modify its appearance.

**Comment by rblackfox** [comment]
Thanks Daveden2, I will close this then. Cheers!

**Comment by nicholaszein** [comment]
Hello @rblackfox,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
This issue is a duplicate of #21466. Please read [my comment on that issue](https://github.com/elementor/elementor/issues/23633#issuecomment-1702979214).

As @Daveden2 kindly pointed out, this is an accessibility feature. In my comment I suggest how to change the color of the outline (but more importantly how to target it). If you want a different behavior, you'll need to use one of the options suggested here and there.

This is the expected behavior of this feature, and we have no plans to change it.

### 👍 For now, that's all we can help with. 

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
